dev/com.ibm.ws.ssl/src/com/ibm/websphere/ssl/Constants.java

```diff
@@ -201,6 +201,7 @@ private Constants() {
     public static final String PROTOCOL_TLSV1_1 = "TLSv1.1";
     public static final String PROTOCOL_TLSV1_2 = "TLSv1.2";
     public static final String PROTOCOL_TLSV1_3 = "TLSv1.3";
+    public static final String PROTOCOL_TLS_FIPS = PROTOCOL_TLSV1_2 + ", " + PROTOCOL_TLSV1_3;
```

Comment:
Based off of the other change in this class do you need a constant for both FIPS 140-2 and 140-3 instead of just FIPS?

Reply:
Probably not since there is no check for FIPS 140-2 enabled

Reply:
The reason I asked was because of the name of the variable.  If it is specific to 140-3, it would be good for it to have that in the name.  It is confusing having just FIPS in a variable name when it is specific to 140-3.